package.json

```diff
@@ -123,7 +133,7 @@
     "@typescript-eslint/eslint-plugin": "^4.14.0",
     "@typescript-eslint/parser": "^4.14.0",
     "electron": "^13.0.1",
-    "electron-builder": "^22.11.3",
+    "electron-builder": "^22.10.5",
```

Comment:
Why was this changed?

Reply:
Because this is the stable version. The other one is beta and is not creating releases automatically.